src/complex.cr

```diff
@@ -36,6 +39,48 @@ struct Complex
     false
   end
 
+  # Returns `self`.
+  def to_c
+    self
+  end
+
+  # Returns the value as a `Float64` if possible (the imaginary part should be exactly zero),
+  # raises otherwise.
+  def to_f64
+    unless @imag.zero?
+      raise Exception.new "The imaginary part should be exactly zero"
```

Comment:
The error message should simply describe what's wrong, not how it should be, for example: `Complex number with non-zero imaginary part can't be converted to real number.`

Reply:
Changed, thanks!